lib/ClusterShell/CLI/Display.py

```diff
@@ -137,6 +151,42 @@ def __init__(self, options, config=None, color=None):
             if hasattr(options, 'debug') and options.debug:
                 self.verbosity = VERB_DEBUG
 
+    def _has_cli_color(self):
+        """Tests CLICOLOR environment variable to determine wether to
+        use color or not on output.
+        """
+        # When CLICOLOR_FORCE is set to something else than 0
+        # colors must be used.
+        if os.getenv("CLICOLOR_FORCE", "0") != "0":
+            return True
+
+        cli_color = os.getenv("CLICOLOR")
+
+        if cli_color is None:
+            return None
+        elif cli_color != "0" and sys.stdout.isatty():
+            # CLICOLOR are ment to be used
+            return True
+        elif cli_color != "0":
+            # CLICOLOR should be used but stdout is not a tty.
+            return False
+        elif cli_color == "0":
+            # CLICOLOR is set to not display colors.
+            return False
+        else:
+            # CLICOLOR is not set
+            return None
+
+    def _has_no_color(self):
+        """Tests NO_COLOR environment variable to determine if color
+        must be muted on output."""
+
+        if os.getenv('NO_COLOR') is not None:
```

Comment:
prefer

    return 'NO_COLOR' in os.environ

And this could be directly use above maybe, no need for a function?